app/src/main/java/com/client/fire_and_water/Player.kt

```diff
@@ -0,0 +1,19 @@
+package com.client.fire_and_water
+
+import java.util.*
+
+class Player {
+    var uuid: Int = 0
+    lateinit var username : String
+
+    var experience : Int = 0
+    var passedLevels : HashMap<Int, Int> = HashMap()
```

Comment:
Doesn't look well
Tell me more about this map

Reply:
supposed that client knows player`s passed levels and points player got for them